.asf.yaml

```diff
@@ -0,0 +1,19 @@
+# https://cwiki.apache.org/confluence/display/INFRA/.asf.yaml+features+for+git+repositories
+
+github:
+  description: "Apache Lucene and Solr open-source search software"
+  homepage: https://lucene.apache.org/
+  labels:
+    - lucene
+    - solr
+    - search
+    - sql
+    - nosql
+    - java
+    - backend
+    - search-engine
```

Comment:
lets add "information-retrieval" like my proposal on the mailing list had?

Do you really think "sql" is a suitable entry in this list?  This is one small feature of Solr and it's _very_ limited to the full syntax people expect from SQL.  So, no, I think we shouldn't put it in this list.
I think "java" is redundant with GitHub's language analysis.

Reply:
Feel free to commit further improvements under the same jira number. I agree that ‘sql’ may be misleading